lib/rules/email.js

```diff
@@ -26,11 +26,11 @@ module.exports = function({ schema, messages, customValidation }, path) {
 	}
 
 	src.push(`
-		if (!${pattern.toString()}.test(value))
+		if (!${pattern.toString()}.test(value)) {
 			${this.makeError({ type: "email",  actual: "value", messages })}
+		}
 
-		${customValidation("value")}
-
+		${this.makeCustomValidator({ path, schema, messages, context })}
```

Comment:
As I see, all rules code contains the same source code. Can we move it into the common parts in validator.js from here? Similar to the required/optional checks which are same in all rules.

Reply:
Yesss!